apps/server/src/modules/task/mapper/task.mapper.spec.ts

```diff
@@ -82,6 +85,38 @@ describe('task.mapper', () => {
 			expect(spy).toHaveBeenCalled();
 			expect(result).toStrictEqual(expected);
 		});
+
+		it('should map task with status, its task card and description values', () => {
+			const taskCard = taskCardFactory.buildWithId();
+
+			const linkedTask = taskCard.task;
+			linkedTask.taskCard = taskCard.id;
```

Comment:
we don't need to call taskCard builder,  because the taskCard is just a string
After all, we want to test the unit, and using other objects / modules should be avoided.

task.taskCard = 'foo' also works instead of all of this.
```suggestion
			const linkedTask = taskFactory.buildWithId({ availableDate: new Date(), dueDate: new Date() });
			task.taskCard = 'foo'
```

and this makes it very similar to the previous test, that I think we should just put it there